src/components/Bundle/Bundle.jsx

```diff
@@ -18,8 +18,15 @@ const Bundle = (props) => {
       as={'div'}
       data-testid={'bundle'}
```

Comment:
```suggestion
      as="div" // Refactor with conditional rendering?
      data-testid="bundle"
```
While looking into this part, why is a button changed to a div? If this is for when there is no `onClick`, don't we still want the button as a button when there is an `onClick`?

Could conditionally render `StyledNakedButton` or `Box` instead and then won't need to update the styles in `NakedButton` which is always a button. e.g. something like below?

`const Container = props.onClick ? StyledNakedButton : Box`

Reply:
This makes sense - the onClick changes were to do with a particular state of bundle that had no button associated with it